v3.1/reference/transactions/overview.md

```diff
@@ -160,3 +160,48 @@ rollback;
 ```
 
 以上例子中，第二条语句执行失败。由于调用了 `rollback`，因此事务不会将任何数据写入数据库。
+
+## 事务大小
+
+对于 TiDB 事务而言，事务太大或者太小，都会影响事务性能。为了克服上述事务在处理过程中的不足，在实际应用中可以根据事务大小进行针对性处理。
+
+### 小事务
+
+在自动提交状态 (`autocommit = 1`) 下，下面三条语句各为一个事务：
+
+```sql
```

Comment:
- Suggest moving the code comment out of the block.
- Add code snippets to make code copyable.

Please also update other code blocks in this PR.